tutorials/epochs/30_epochs_metadata.py

```diff
@@ -88,7 +88,6 @@
 # groups.
 
 epochs.metadata["NumberOfLetters"] = epochs.metadata["NumberOfLetters"].map(int)
-
```

Comment:
touched it for rendering

Reply:
Oddly enough looking at that example I don't think `epochs` is ever just printed, but I confirmed locally that it works
<img width="515" alt="Screenshot 2024-06-29 at 3 27 26 PM" src="https://github.com/mne-tools/mne-python/assets/2365790/743048b8-a112-42a9-a7be-84b0648dcbe6">